frankly-js/FranklyClient.html

```diff
@@ -73,7 +73,7 @@ <h4 class="name" id="FranklyClient"><span class="type-signature"></span>new Fran
 with the same or different pairs of appKey and appSecret.
 
 Each instance of FranklyClient maintains its own connection pool to Frankly
-servers, if the client is not required anymore then the application should
+servers, if the client is not required anymore then the application should 
```

Comment:
I guess the simplest way to fix that is to make a new branch from master and cherry-pick the new files that need to be included.